testing/testcafe/tests/scheduler/api/deleteRecurrence.ts

```diff
@@ -106,10 +105,10 @@ safeSizeTest('should exclude from recurrence if mode is "occurrence"', async (t)
       apptColor: 1,
       recurrenceRule: 'FREQ=DAILY;COUNT=4',
     }],
-  },
+  }, true,
 ));
 
-safeSizeTest('should show delete recurrence dialog if mode is "dialog"', async (t) => {
+test.skip('should show delete recurrence dialog if mode is "dialog"', async (t) => {
```

Comment:
Should this test be skipped?

Reply:
Skipped this, I think we shouldn't skip this test

Let's bring it back :)

Reply:
It should be skipped. The test creates two instances of dialog instead of one. I discussed this test with @andrewmakarov and we decided to skip it. Andrew promises to investigate the issue and delete skip after that.